src/lib.rs

```diff
@@ -11,6 +9,9 @@ use slog::{error, o, warn, Logger};
 use settings::Settings;
 mod settings;
 
+const POD_SPEC_MUTATE: bool = true;
+const POD_SPEC_NOT_MUTATE: bool = false;
```

Comment:
I've added this constants to avoid "magic numbers". I found more readable to have meaningful name in the later `match` then the boolean values `true/false`.

Reply:
I like the idea, but I would change the names to be:

- POD_SPEC_MUTATED
- POD_SPEC_NOT_MUTATED

Reply:
Or event better, change that to become an Enum, this is even more clear than using a boolean.

Rust types for the win!